packages/transport-webrtc/package.json

```
@@ -38,18 +38,20 @@
     "test": "aegir test -t browser",
     "test:chrome": "aegir test -t browser --cov",
     "test:firefox": "aegir test -t browser -- --browser firefox",
+    "test:webkit": "aegir test -t browser -- --browser webkit",
     "lint": "aegir lint",
     "lint:fix": "aegir lint --fix",
     "clean": "aegir clean",
-    "dep-check": "aegir dep-check"
+    "dep-check": "aegir dep-check -i protons"
```

This is already in the [default ignore list](https://github.com/ipfs/aegir/blob/369453f7543488d43e72721c48320d6d8d59b705/src/dependency-check.js#L13)

```suggestion
    "dep-check": "aegir dep-check"
```